CONTRIBUTING.md

```diff
@@ -60,6 +60,9 @@ You can find all available build versions in the top level pom.xml file. If you
 for Databricks then you should use the `jenkins/databricks/build.sh` script and modify it for
 the version you want.
 
+See the [scala2.13](scala2.13) directory for instructions on how to build against 
```

Comment:
might also be nice to state that the code must compile against both versions so when developing keep that in mind. Not sure if its useful to mention a few common things that arise.  Also might point to the 2.13/2.12 source subdirectories

Reply:
Added some documentation updates, please let me know if those make sense.